manage.py

```diff
@@ -46,10 +53,24 @@ def test(all=False):
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
 
+    q = Queue('default', connection=conn)
     while True:
         logging.info("Starting scrape...")
-        scrape_and_categorize_articles()
-        logging.info("Finished scrape. Let's run it back...")
+        job = q.enqueue(scrape_and_categorize_articles)
```

Comment:
I think we should raise that as a second issue as a more efficient way to do this. Don’t think it strictly needs to be part of this PR (given that there will almost certainly be rq bugs related to that). Can you raise that? 